data/resistance intro.txt

```diff
@@ -0,0 +1,1187 @@
+		not "Heliarch Investigation 2: active"
+		not "Heliarch Containment 4-A: offered"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		conversation
+			`Once your ship lands, your monitor screen twitches, and shows a message rapidly forming.`
+			`	"Captain, we're in urgent need of ships willing to evacuate some comrades. On <stopovers>, Heliarch agents are looking all over for our whereabouts, and have already stumbled upon some of our factories.`
+			`	"Greatly compensated, you will be. Please help us Captain."`
+			choice
+				`	(Say you will be heading there soon.)`
+					accept
+				`	(Say you can't risk it.)`
+					decline
+	on stopover
+		dialog `With Heliarch ships flying all over the planet, you start to wonder if you're too late as you come in for a landing. The moment you land, however, a message pops up again, telling you to open your hatch. You do, and the 35 passengers enter your ship, urging you to leave quickly.`
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		"assisted resistance" ++
+		payment 3421957
+			`Upon landing, several of the Resistance members run up to you in the cockpit, giving their thanks. Some also insist on hugging you, and though it proves rather difficult with the Kimek and Arachi of the group, they don't seem to mind.`
+			`	The last one, a young Saryd, hands you <payment> after thanking you.`
+			`	"The total sum of what we all could bring, that is. For saving us, hopefully it suffices."`
+			choice
+				`	"How were you found?"`
+					goto how
+				`	"What would've happened if you got caught?"`
+					goto caught
+			label how
+			`	"Spies, the Heliarch use. Among civilians or even our own, they sometimes hide," she says.`
+			`	They all thank you once again for bailing them out, and disappear in the endless masses of people mingling here.`
+				accept
+			`label caught`
+			`	"Imprisoned, sometimes some are, when caught they are," she says, not wanting to disclose much more about the "sometimes", and as you see the reaction of the others to the statement, you decide not to press for any more details.`
+			`	They all thank you once again for bailing them out, and disappear in the endless masses of people mingling here.`
+
+
+
+mission "Heliarch Interrogation 1"
+	name "Meet with the Heliarch"
+	description "Head to <destination> by <date>, so that the Heliarch can ask you some questions."
+	landing
+	deadline
+	source
+		government "Coalition"
+	destination
+		government "Heliarch"
+	to offer
+		has "Resistance: Evacuation: done"
+		not "Resistance: Questions: offered"
+	on offer
+		conversation
+			`Upon landing, a Heliarch ship lands close by, and hails you.`
+			`	"To <destination>, you are to go immediately. Asked some questions, you will be. Until <date>, to go there you have."`
+			`	They cut off the transmission, and head back up to space.`
+				accept
+	on fail
+		"reputation: Heliarch" = -1000
+		"reputation: Coalition" = -1000
```

Comment:
Maybe add a dialog that mentions the effects of failing the mission?

Reply:
what should the dialog say?

Reply:
`You receive a message from the Heliarchs. "Captain <first> <last>. Arrived at <destination>, you have not. Run out, the allocated time has. Forced to treat you as a criminal, we are. Regrettable, your choice is."`

Reply:
what if you get the mission in Arach Space, and JD as close as you can to Earth though? what I thought would be a more passable version would be something where you take a look at the date and back at your map, but the whole thing could easily just turn into a big wall in the middle of the screen if we're to explain it properly. it might be best to just have no dialog